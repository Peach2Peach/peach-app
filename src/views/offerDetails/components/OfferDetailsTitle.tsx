import { Text } from '../../../components'
import i18n from '../../../utils/i18n'
import { getHeaderStyles } from '../../../utils/layout'
import { offerIdToHex } from '../../../utils/offer'

type Props = {
  id: string
}

export const OfferDetailsTitle = ({ id }: Props) => {
  const { fontSize } = getHeaderStyles()
  return (
    <Text style={fontSize} numberOfLines={1}>
      {i18n('offer')} {offerIdToHex(id)}
    </Text>
  )
}
