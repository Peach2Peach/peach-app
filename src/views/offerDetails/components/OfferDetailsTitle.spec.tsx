import { createRenderer } from 'react-test-renderer/shallow'
import { OfferDetailsTitle } from './OfferDetailsTitle'

describe('OfferDetailsTitle', () => {
  const shallowRenderer = createRenderer()
  it('should render correctly', () => {
    shallowRenderer.render(<OfferDetailsTitle id="123" />)
    expect(shallowRenderer.getRenderOutput()).toMatchSnapshot()
  })
})
