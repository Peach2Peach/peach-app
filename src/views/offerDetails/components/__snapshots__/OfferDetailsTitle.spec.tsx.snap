// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferDetailsTitle should render correctly 1`] = `
<PeachText
  numberOfLines={1}
  style={
    {
      "fontFamily": "Baloo2-Bold",
      "fontSize": 16,
      "letterSpacing": 0.16,
      "lineHeight": 26,
    }
  }
>
  offer
   
  Pâ€‘7B
</PeachText>
`;
